README.md

```diff
@@ -1404,7 +1404,6 @@ _Related: [Wikis](#wikis)_
 - [mkaczanowski pastebin](https://github.com/mkaczanowski/pastebin) - Simple, fast, feature-rich, standalone pastebin service. `MIT` `Rust`
 - [mojopaste](https://metacpan.org/dist/App-mojopaste) - Perl based pastebin. ([Demo](https://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `Artistic-2.0` `Perl`
 - [MokinToken](https://github.com/nexus-uw/mokintoken) - Clientside encrypted pastebin using tweetnacl. `Unlicense` `PHP`
-- [NoteHub](https://github.com/chmllr/NoteHub) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. `MIT` `Nodejs`
```

Comment:
Maybe NoteHub should not be removed? It looks like a very simple codebase which may not need much improvements/maintenance:

```
$ cloc NoteHub/
      21 text files.
      21 unique files.                              
       5 files ignored.

github.com/AlDanial/cloc v 1.86  T=0.30 s (56.1 files/s, 4980.4 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Go                               6            105              8            804
CSS                              1             52              1            229
HTML                             4              0              0            138
Markdown                         2             29              0             78
JavaScript                       2              5              0             44
TOML                             1              1              0              6
make                             1              2              0              6
-------------------------------------------------------------------------------
SUM:                            17            194              9           1305
-------------------------------------------------------------------------------
```

@chmllr do you still use/maintain NoteHub?

Reply:
No I don't.